test-data/unit/check-inline-config.test

```diff
@@ -157,3 +157,12 @@ main:4: error: Unterminated quote in configuration comment
 # mypy: skip-file
 [out]
 main:1: error: Unrecognized option: skip_file = True
+
+[case testIgnoreErrorCodesInline]
+# mypy: ignore-error-codes="attr-defined,arg-type"
+
+not_defined  # E: Name 'not_defined' is not defined
+x: int = 'no'  # E: Incompatible types in assignment (expression has type "str", variable has type "int")
+'no'.no_way
+def test(x: str) -> None: ...
```

Comment:
Also add daemon test case (to make sure the daemon applies these after updates).